### PR TITLE
Threat matrix UI changes

### PR DESCRIPTION
**Pull request by satyambnsal**
**Changes Summary**
Updated Wanted Indicator from threat percentage to threat matrix High, Med, and low.
logic for threat matrix:
if threat percentage is >= 85 : High Threat
else if if threat percentage is >= 29 : Medium Threat
else Low Threat

Relevant Issue: #305

### CODE REVIEW

**Comment by satyambnsal** [comment]
@notV4l

File: web/src/components/player/WantedIndicator.tsx [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
-export const WantedIndicator = ({ wanted, ...props }: { wanted: number } & StyleProps) => {
+enum ThreatLevels {
+  High = "HIGH",
+  Medium = "MED",
+  Low = "LOW",
+}
+
+export const WantedIndicator = ({
+  wanted,
+  max_wanted_shopping,
```
Comment (notV4l) [inline web/src/components/player/WantedIndicator.tsx:15]: lets call it highLimit or smth more generic & camelCased

File: web/src/components/player/WantedIndicator.tsx [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
+  ...props
+}: { wanted: number; max_wanted_shopping: number } & StyleProps) => {
+  const wantedTick = Math.floor((wanted * 7) / 100);
+  const threatLevel =
```
Comment (notV4l) [inline web/src/components/player/WantedIndicator.tsx:19]: you can get wantedTick with game.wanted.getWantedTick( locationId
ticks are int so doesnt really make sense to test >=4.66 >=2.33

Comment (satyambnsal) [reply web/src/components/player/WantedIndicator.tsx:18]: Ah nice, i didn't knew that. changes done. Thank you!

**Pull request merged**